src/puzzle-latency/puzzle-latency.css

```diff
@@ -11,3 +11,18 @@
   line-height: 1;
   vertical-align: middle;
 }
+.pm-PuzzleLatency-value-green {
```

Comment:
This looks good and it works.  Thanks!

I might change it a bit in the future to use the design tokens and get the colors and such from there.